collections/reverse-engineering-tools/index.md

```diff
@@ -0,0 +1,25 @@
+---
+items:
+  - ReFirmLabs/binwalk
+  - codilime/veles
+  - kaitai-io/kaitai_struct
+  - desowin/usbpcap
+  - icsharpcode/ILSpy
+  - radare/radare2
+  - radareorg/cutter
+  - Storyyeller/Krakatau
+  - deathmarine/Luyten
+  - java-decompiler/jd-gui
+  - 0xd4d/de4dot
+  - avast-tl/retdec
+  - yegord/snowman
+  - x64dbg/x64dbg
+  - mitmproxy/mitmproxy
+  - ohjeongwook/DarunGrim
+  - dbeaver/dbeaver
+  - lucasg/Dependencies
+  - aquynh/capstone
+  - beautify-web/js-beautify
+display_name: Reverse Engineering Tools
+---
+Get knowledge for fun and profit!
```

Comment:
@KOLANICH Would still like to see it be a bit longer. https://en.wikipedia.org/wiki/Reverse_engineering#Reverse_engineering_of_software may provide some inspiration. Just a sentence or two would be good, thanks.

Reply:
I don't know what can be added without spoiling the laconicity.

Reply:
The laconicity is a bug, not a feature 😉 

What about something like:

> Tools to aid with the reverse engineering of binary, minified or compiled computer programs or protocols.

Reply:
1 tautology
2 not very correct. What is "minified or compiled  <...> protocols"?

Reply:
@KOLANICH Please feel free to reword. I'm not really interested in arguing about this; let's either find something better or close this PR.